recipes/alphashape/LICENSE

```diff
@@ -0,0 +1,14 @@
+The MIT License (MIT)
+Copyright � 2019 Lilian Besson (Naereen), https://bitbucket.org/lbesson/ <naereen at crans dot org>
```

Comment:
Looks like the license file is part of the pypi tarball, so please remove it from here.

Reply:
removed.